caa-redirect/recipes/default.rb

```diff
@@ -63,13 +61,14 @@
       EOH
     action :nothing
     subscribes :run, "git[/home/caaredirect/#{service_name}]"
+    notifies :restart, "daemontools_service[#{service_name}]"
   end
 
   daemontools_service "#{service_name}" do
     directory "/home/caaredirect/svc-#{service_name}"
     template "coverart_redirect"
     log true
-    action [:enable, :start]
+    action [:enable, :up]
   end
 
   link "/home/caaredirect/svc-#{service_name}/coverart_redirect" do
```

Comment:
No, I think that's the only use. Using a template variable makes more sense. :+1: